client/templates/find_certificate.html

```diff
@@ -12,6 +12,7 @@
         <form action="{% url 'client_find_certificate' %}" method="post">
             {% csrf_token %}
             {{ form|bootstrap }}
+            רק משתמשים רשומים במערכת זכאים להנפיק תעודה <br>
```

Comment:
* Move the text to the top of the form, so it's clearer
* The volunteers aren't "משתמשים", and a link to the volunteer form would be nice too. I'd change the text to "עוד לא נרשמתם כמתנדבים? לחצו כאן" - this makes it clear this form is only for registered volunteers
 * Wrap the text in <p> tag so it's not floating, maybe enlargen the text a little.